blogs/tekton-oadp-nonadmin/install_templates/templates/03-rbac-pipeline-role.yaml

```diff
@@ -17,9 +23,21 @@ objects:
     name: oadp-nonadmin-role
   rules:
   - apiGroups:
+    # By default restore resources will NOT be provided by this template.
+    # It is recommended that restores are executed by OpenShift Admin's in production environments.
+    # For demonstration purposes only you may uncomment the restore resources.
     - velero.io
     resources:
-    - '*'
+    - 'backuprepositories'
+    - 'backups'
+    - 'deletebackuprequests'
```

Comment:
Probably just want admins , some one may be able to delete someone else's?

Reply:
meh.. it's just a guide.. so I'll remark it out